src/kOS/Suffixed/StageValues.cs

```diff
@@ -1,28 +1,54 @@
 ﻿using System;
+using System.Linq;
 using kOS.Safe.Encapsulation;
-using kOS.Utilities;
+using kOS.Safe.Encapsulation.Suffixes;
 
 namespace kOS.Suffixed
 {
     public class StageValues : Structure
     {
-        private readonly Vessel vessel;
+        private readonly SharedObjects shared;
 
-        public StageValues(Vessel vessel)
+        public StageValues(SharedObjects shared)
         {
-            this.vessel = vessel;
+            this.shared = shared;
+
+            InitializeSuffixes();
+        }
+
+        private void InitializeSuffixes()
+        {
+            AddSuffix("RESOURCES", new Suffix<SuffixedList<ActiveResourceValue>>(GetResourceManifest));
```

Comment:
If we want to keep STAGE:RESOURCES i would like to add SHIP:RESOURCES to keep them looking the same